Core/Vault.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Core;
+
+use DateInterval;
+use DateTime;
+
+use OxidEsales\Eshop\Core\Database\Adapter\DatabaseInterface;
+use OxidEsales\Eshop\Core\DatabaseProvider;
+use OxidEsales\Eshop\Core\Exception\DatabaseConnectionException;
+use OxidEsales\Eshop\Core\Exception\DatabaseErrorException;
+use OxidEsales\Eshop\Core\Registry;
+
+use Wirecard\PaymentSdk\Response\SuccessResponse;
+
+/**
+ * Vault for saving credit card information
+ *
+ * @since 1.3.0
+ */
+class Vault
+{
+    /**
+     * Get all cards associated with the user and address
+     *
+     * @return array of cards with keys 'OXID', 'USERID', 'ADDRESSID', 'TOKEN', 'MASKEDPAN', 'EXPIRATIONMONTH',
+     *         'EXPIRATIONYEAR'
+     *
+     * @throws DatabaseConnectionException
+     *
+     * @since 1.3.0
+     */
+    public static function getCards()
+    {
+        $oUser = Registry::getSession()->getUser();
+        $sUserId = $oUser->getId();
+        $sAddressId = $oUser->getSelectedAddressId();
+        $aCards = [];
+        try {
+            $sQuery = "SELECT * from " . OxidEeEvents::VAULT_TABLE . " " .
```

Comment:
The string concatenation at the end is not necessary when using double quotes.
```suggestion
            $sQuery = "SELECT * from " . OxidEeEvents::VAULT_TABLE . " 
```

Reply:
👍